tools/mslite

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+bold() {
+    echo "$(tty -s && tput bold)$1$(tty -s && tput sgr0)"
+}
+
+clean() {
+    bold "mindspore clean"
+    rm -rf "./third_party/source/mindspore"
+}
+
+sync() {
+    bold "mindspore sync"
+    [ -d "./third_party/source/mindspore" ] || git clone --quiet https://github.com/mindspore-ai/mindspore.git "./third_party/source/mindspore"
```

Comment:
Instead of cloning the entire repo only `curl` the `.fbs` files. See `./tools/paddle`.